vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -790,7 +790,7 @@ - (void)refreshPublisherWithId:(NSString *)publisherId completion:(void (^)(BATP
 - (void)processSKUItems:(NSArray<BATSKUOrderItem *> *)items
              completion:(void (^)(BATResult result, NSString *orderID))completion
 {
-  ledger->ProcessSKU(VectorFromNSArray(items, ^ledger::type::SKUOrderItem(BATSKUOrderItem *item) {
+  ledger->ProcessSKU(VectorFromNSArray(items, ^ledger::type::SKUOrderItemPtr(BATSKUOrderItem *item) {
     return *item.cppObjPtr;
```

Comment:
The returned value of this is the dereferenced ptr object, so no need to make it `SKUOrderItemPtr`, can just stay as `SKUOrderItem`

Reply:
resolved, thanks 👍